CONTRIBUTING.md

```diff
@@ -1,5 +1,74 @@
-# CONTRIBUTING
+# Contributing to Mistral AI Client for Dart
 
-## Development flow
+Thank you for considering contributing to [Your Project Name]! We appreciate your time and effort.
 
-## How to build
+## Table of Contents
+
+- [Contributing to Mistral AI Client for Dart](#contributing-to-mistral-ai-client-for-dart)
+  - [Table of Contents](#table-of-contents)
+  - [How Can I Contribute?](#how-can-i-contribute)
+    - [Reporting Bugs](#reporting-bugs)
+    - [Suggesting Enhancements](#suggesting-enhancements)
+    - [Code Contributions](#code-contributions)
+    - [Documentation](#documentation)
+  - [Getting Started](#getting-started)
+    - [Setting Up the Development Environment](#setting-up-the-development-environment)
+  - [Style Guide](#style-guide)
+  - [Review Process](#review-process)
+
+## How Can I Contribute?
+
+### Reporting Bugs
+
+Before submitting a bug report, please check if the issue has already been reported in the [issue tracker](https://github.com/nomtek/mistralai_client_dart/issues). If it hasn't, create a new issue with a clear title and description, including steps to reproduce the bug.
+
+### Suggesting Enhancements
+
+If you have an idea for an improvement or a new feature, we'd love to hear about it! Create a new issue in the [issue tracker](https://github.com/nomtek/mistralai_client_dart/issues) with your suggestion, and we can discuss it further.
+
+### Code Contributions
+
+We welcome contributions to the codebase. To contribute:
+
+1. [Fork the repository](https://help.github.com/en/github/getting-started-with-github/fork-a-repo).
+2. Create a new branch for your feature or bug fix.
+3. Make your changes and commit them with clear and concise messages.
+4. Push your changes to your fork.
+5. [Submit a pull request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/proposing-changes-to-your-work-with-pull-requests), explaining the changes you made.
+
+### Documentation
+
+Improving documentation is a valuable contribution. If you find areas that need clarification or have ideas for better explanations, please make the necessary updates.
```

Comment:
> please make the necessary updates

But how should they do it? 🤔 I guess they can't simply edit the code without necessary permissions, right? We should explain to them whether they have to create some issues with documentation fixes or if they can do it themselves somehow 🤔

Reply:
Thx for the suggestion :)

Sorry, I assumed it would go the same way as standard code change. But, as you asked, it's probably not so obvious. I'll try to add some guidance on how to improve documentation.

Reply:
Updated in https://github.com/nomtek/mistralai_client_dart/pull/24/commits/42354570f194dd5e88f3982f0b824e9e1bbca23b